### ISSUE TITLE
Can't send messages to remote SMTP

### ISSUE THREAD

**Issue by alnitac** [nextcloud/mail#10217]
### Steps to reproduce

1. Messages are received but not send
2. Configure smtp connection SSL/TLS with server ssl0.ovh.net port 465, user and password. Authentication OK.
3. Message send appears
4. 5 seconds later, new message "Impossible d'envoyer le message"
5. Installation is Debian, Docker and latest Nextcloud (30)
6. The server ssl0.ovh.net can be joined from nextcloud container on port 465
7. Activated debug, loglevel 0
8. When sending found this message in log : "message":"OCA\\Mail\\Controller\\OutboxController::send uses the @NoAdminRequired annotation and should use the #[OCP\\AppFramework\\Http\\Attribute\\NoAdminRequired] attribute instead"

### Expected behavior

Message shoul be send

### Actual behavior

Messge not send

### Mail app version

4.0.1

### Mailserver or service

OVH

### Operating system

Debian, Docker

### PHP engine version

PHP 8.2

### Web server

Apache (supported)

### Database

PostgreSQL

### Additional info

Nextcloud installed yesterday.

**Comment by ChristophWurst** [comment]
Inspect the data/nextcloud.log file and find the entry for the failing send operation. Post the JSON here.

**Comment by alnitac** [comment]
This seems to be related to the problem

{"reqId":"cEJfVXjRRQJFReIvoQ1J","level":3,"time":"2024-10-02T11:30:38+00:00","remoteAddr":"93.23.95.134","user":"CMC","app":"PHP","method":"GET","url":"/apps/mail/api/avatars/url/MAILER-DAEMON%40mo503.mail-out.ovh.net","message":"file_get_contents(https://mo503.mail-out.ovh.net/): Failed to open stream: Connection refused at /var/www/html/custom_apps/mail/lib/Vendor/Favicon/DataAccess.php#15","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:130.0) Gecko/20100101 Firefox/130.0","version":"30.0.0.14","data":{"app":"PHP"}}

**Comment by ChristophWurst** [comment]
That's the request for an avatar

`tail -f data/nextcloud.log`, reproduce and watch out for the error written when sending fails

**Comment by ChristophWurst** [comment]
it now looks like the message was sent without an error. was there even a SMTP error?

**Comment by alnitac** [comment]
I did not see any error. The smtp host is my provider's (OvhCloud). 
Does it help if i try to send through a local SMTP ?

**Comment by ChristophWurst** [comment]
I think it's best if you reach out to your provider. If there is no SMTP error then the server accepts the data. There is nothing more we can do.

**Comment by alnitac** [comment]
Ok, thank you. I ask my provider.
Still, why do i get the message "Impossible d'envoyer le message" if there is no SMTP error ?

**Comment by ChristophWurst** [comment]
I don't know

Your provided logs do not contain `Could not send message`

If the error only happens occasionally, provide logs for a scenario where you see the message

**Comment by alnitac** [comment]
The error appears every time on the screen, no mail can be send this way. 
I tried the same mail box with RoundCube webmail from OVH and it works Ok.
Incoming mail is also synchronized.
...
I tried with Thunderbird, same problem.
Ovhcloud does not accept an incident because the webmail is operational.
Found some users with similar problem on user forum.
...
Thank you very much for your help.
I'm going to look for another mail solution.
